index.html

```diff
@@ -1102,21 +1086,68 @@
           modifiers for particular payment method identifiers. For example, it
           allows you to adjust the total amount based on payment method.
         </dd>
+      </dl>
+    </section>
+    <section data-dfn-for="PaymentDetailsInit" data-link-for=
+    "PaymentDetailsInit">
+      <h2>
+        <dfn>PaymentDetailsInit</dfn> dictionary
+      </h2>
+      <pre class="idl">
+        dictionary PaymentDetailsInit : PaymentDetailsBase {
+          required PaymentItem total;
```

Comment:
Seems WebIDL should be revised, per [section Dictionaries, WebIDL level 1](https://www.w3.org/TR/WebIDL-1/#idl-dictionaries):
> The identifier of a dictionary member MUST NOT be the same as that of another dictionary member defined on the dictionary or on that dictionary’s inherited dictionaries.